src/bp/ui-studio/src/web/translation/index.tsx

```diff
@@ -0,0 +1,34 @@
+import axios from 'axios'
+import { PrimitiveType } from 'intl-messageFormat'
+import { createIntl, createIntlCache, IntlShape } from 'react-intl'
+
+const defaultLocale = 'en'
+let locale: string
+let intl: IntlShape
+const cache = createIntlCache()
+
+const getUserLocale = () => {
+  const code = str => str.split('-')[0]
+  const browserLocale = code(navigator.language || navigator['userLanguage'] || '')
+  return browserLocale
+}
```

Comment:
We'll need a way to select language for Botpress. This uses the browser language as a temporary measure.